### PR TITLE
Adding overlay for minio artifacts

### PR DESCRIPTION
**Pull request by DharmitD**
- Added minio artifacts into a separate overlay
- Updated the image to have it on `opendatahub/minio` repository on `quay` instead of `gcr`
- Deployed on a test cluster locally to check that the changes are applied correctly and the resources come up as expected.

### CODE REVIEW

File: kfdef/kfctl_openshift_ml-pipelines.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ spec:
         overlays:
           - metadata-store-mysql
           - ml-pipeline-ui
+          - minio
```
Comment (rimolive) [inline kfdef/kfctl_openshift_ml-pipelines.yaml:17]: Better name would be `object-store-minio`

**Review by rimolive — CHANGES_REQUESTED** [review]

File: kfdef/kfctl_openshift_ml-pipelines.yaml (hunk 1 above)
Comment (DharmitD) [reply kfdef/kfctl_openshift_ml-pipelines.yaml]: Done, updated name to be `object-store-minio`

**Comment by rimolive** [comment]
You need to reference this overlay in README too: https://github.com/anishasthana/odh-manifests/blob/propose_pipelines/ml-pipelines/README.md#overlays

**Comment by DharmitD** [comment]
> You need to reference this overlay in README too: https://github.com/anishasthana/odh-manifests/blob/propose_pipelines/ml-pipelines/README.md#overlays

Done, updated the README to have a reference to this overlay.

**Comment by DharmitD** [comment]
Closing this PR, since it is a duplicate of https://github.com/opendatahub-io/odh-manifests/pull/652

**Pull request closed without merging**